scripts/preload/win32_wrap.rb

```diff
@@ -0,0 +1,369 @@
+			def initialize
+				@index = 0
+			end
+			def call(args)
+				seq = [args[0], args[1], args[2], args[3]]
+
+				if seq == Seq[@index] or seq == Seq2[@index]
+					@index += 1
+				else
+					@index = 0
+				end
+
+				if @index == 4
+					@index = 0
+					Graphics.fullscreen = !Graphics.fullscreen
+				end
+			end
+		end
+
+		class GetKeyState
+			def call(vkey)
+				return common_keystate(vkey[0])
+			end
+		end
+		class GetAsyncKeyState
+			PRESSED_BIT = (1 << 15)
+			def call(vkey)
+				return common_keystate(vkey[0]) == 1 ? PRESSED_BIT : 0
+			end
+		end
+		class GetKeyboardState
+			PRESSED_BIT = 0x80
+			def call(args)
+				out_states = args[0]
+				states = get_raw_keystates
+
+				Scancodes::WIN32.each do |name, val|
+					pressed = false
+
+					if name == :LBUTTON
+						pressed = Input.press?(Input::MOUSELEFT)
+					elsif name == :RBUTTON
+						pressed = Input.press?(Input::MOUSERIGHT)
+					elsif name == :MBUTTON
+						pressed = Input.press?(Input::MOUSEMIDDLE)
+					elsif name == :SHIFT
+						pressed = double_state(states, :LSHIFT, :RSHIFT)
+					elsif name == :MENU
+						pressed = double_state(states, :LALT, :RALT)
+					elsif name == :CONTROL
+						pressed = double_state(states, :LCONTROL, :RCONTROL)
+					else
+						scan = nil
+						if Scancodes::SDL.key?(name)
+							scan = name
+						else
+							scan = Scancodes::WIN2SDL[name]
+						end
+
+						pressed = state_pressed(states, name)
+					end
+
+					out_states.setbyte(val, pressed ? PRESSED_BIT : 0)
+				end
+				return 1
+			end
+		end
+
+		class ShowCursor
+			def initialize
+				@cursor_count = 0
+			end
+			def call(args)
+				if args[0] == 1
+					@cursor_count += 1
+				else
+					@cursor_count -= 1
+				end
+
+				Graphics.show_cursor = @cursor_count >= 0
+			end
+		end
+
+		class GetCursorPos
+			def call(args)
+				out = [Input.mouse_x, Input.mouse_y].pack('ll')
+				memcpy_string(args[0], out)
+				return 1
+			end
+		end
+
+		class GetClientRect
+			def call(args)
+				return 0 if args[0] != 42
+				rect = [0, 0, 640, 480]
+				begin
+					rect[2] = Graphics.width
+					rect[3] = Graphics.height
+				rescue
+				end
+				memcpy_string(args[1], rect.pack('l4'))
+				return 1
+			end
+		end
+
+		class ScreenToClient
+			def call(args)
+				return 1
+			end
+		end
+
+		class FindWindowA
+			def call(args)
+				if args[0] == "RGSS Player"
+					return 42
+				else
+					return 0
+				end
+			end
+		end
+	end
+end
+
+def kappatalize(s)
+	s[0] = s[0].upcase
+	return s
+end
+
+class Win32API
+	def initialize(dll, func, *args)
+		@dll = dll
+		@func = func
+		@called = false
+
+		dll = kappatalize(dll.chomp(".dll"))
+		func = kappatalize(func)
+
+		if Win32API_Impl.const_defined?(dll)
+			dll_impl = Win32API_Impl.const_get(dll)
+			if dll_impl.const_defined?(func)
+				@impl = dll_impl.const_get(func).new
+			end
+		end
+
+	end
+	def call(*args)
+		if @impl
+			return @impl.call(args)
+		end
+
+		MKXP.puts("[#{@dll}:#{@func}] #{args.to_s}") if !@called
```

Comment:
```suggestion
		System.puts("[#{@dll}:#{@func}] #{args.to_s}") if !@called
```
May as well remove that last MKXP reference to make this script standalone.

That should be everything.